pkg/apis/pipeline/v1/param_types_test.go

```diff
@@ -369,23 +369,23 @@ func TestArrayOrString_UnmarshalJSON(t *testing.T) {
 	}
 }
 
-func TestArrayOrString_UnmarshalJSON_Directly(t *testing.T) {
+func TestParamValues_UnmarshalJSON_Directly(t *testing.T) {
 	cases := []struct {
 		desc     string
 		input    string
-		expected v1.ArrayOrString
+		expected v1.ParamValue
 	}{
-		{desc: "empty value", input: ``, expected: *v1.NewArrayOrString("")},
-		{desc: "int value", input: `1`, expected: *v1.NewArrayOrString("1")},
-		{desc: "int array", input: `[1,2,3]`, expected: *v1.NewArrayOrString("[1,2,3]")},
-		{desc: "nested array", input: `[1,\"2\",3]`, expected: *v1.NewArrayOrString(`[1,\"2\",3]`)},
-		{desc: "string value", input: `hello`, expected: *v1.NewArrayOrString("hello")},
-		{desc: "array value", input: `["hello","world"]`, expected: *v1.NewArrayOrString("hello", "world")},
+		{desc: "empty value", input: ``, expected: *v1.NewStructuredValues("")},
+		{desc: "int value", input: `1`, expected: *v1.NewStructuredValues("1")},
+		{desc: "int array", input: `[1,2,3]`, expected: *v1.NewStructuredValues("[1,2,3]")},
+		{desc: "nested array", input: `[1,\"2\",3]`, expected: *v1.NewStructuredValues(`[1,\"2\",3]`)},
+		{desc: "string value", input: `hello`, expected: *v1.NewStructuredValues("hello")},
+		{desc: "array value", input: `["hello","world"]`, expected: *v1.NewStructuredValues("hello", "world")},
 		{desc: "object value", input: `{"hello":"world"}`, expected: *v1.NewObject(map[string]string{"hello": "world"})},
 	}
 
 	for _, c := range cases {
-		aos := v1.ArrayOrString{}
+		aos := v1.ParamValue{}
```

Comment:
nit: Can we have the related variable naming changed, as they are previously for `ArrayorString`?

Reply:
thanks! Didn't realize we have aos